sim/models/iris_depth_camera_3/iris_depth_camera.sdf

```diff
@@ -148,7 +122,7 @@
           <cameraInfoTopicName>rgb/camera_info</cameraInfoTopicName>
           <depthImageTopicName>depth/image_raw</depthImageTopicName>
           <depthImageCameraInfoTopicName>depth/camera_info</depthImageCameraInfoTopicName>
-          <pointCloudTopicName>depth/points</pointCloudTopicName>
+          <pointCloudTopicName>depth_registered/points</pointCloudTopicName>
```

Comment:
why did you change the namespace? This brakes https://github.com/PX4/avoidance/blob/master/local_planner/src/nodes/local_planner_node.cpp#L161-L164 which expects `*/points` and `*/camera_info`to be in the same namespace.

Reply:
ok, sure. I changed it because one of the launch files would not work with those topic names. But now I changed it the other way round such that the launch file take depth/points as an input